ocrd_utils/ocrd_utils/logging.py

```diff
@@ -46,19 +46,23 @@
     'setOverrideLogLevel',
 ]
 
+# These are the loggers we add handlers to
+ROOT_OCRD_LOGGERS = [
+    'ocrd',
+    'ocrd_network'
+]
+
 LOGGING_DEFAULTS = {
     'ocrd': logging.INFO,
-    'ocrd_models': logging.INFO,
-    'ocrd_utils': logging.INFO,
-    'ocrd_network': logging.INFO,
-    'ocrd_exif': logging.INFO,
+    'ocrd_network': logging.DEBUG,
     # 'ocrd.resolver': logging.INFO,
     # 'ocrd.resolver.download_to_directory': logging.INFO,
     # 'ocrd.resolver.add_files_to_mets': logging.INFO,
     # To cut back on the `Self-intersection at or near point` INFO messages
     'shapely.geos': logging.ERROR,
     'tensorflow': logging.ERROR,
     'PIL': logging.INFO,
+    'paramiko.transport': logging.INFO
 }
```

Comment:
The FastAPI and Uvicorn loggers are not reflected here. This leads to their log suppression - I no longer see output from them.

I think that should work and be enough:
```python
'uvicorn.access': logging.DEBUG,
'uvicorn.error': logging.DEBUG,
'uvicorn': logging.INFO
```
Have a look here: https://github.com/tiangolo/fastapi/issues/1508

Reply:
I only copied those from the hard-coded definition in `ocrd/ocrd/cli/network.py`, feel free to adapt the way you see fit.

Reply:
That suggestion did not fix it.

Reply:
And you're using the `ocrd_logging.conf`? These must be set in the config file as well, the `LOGGING_DEFAULTS` are only set for the builtin logging (i.e. no ocrd_logging.conf).

If that does not help, it might be dependant on when those loggers are first accessed. Can you try

```python.diff
-    if config_file:                                                            
-        logging.config.fileConfig(config_file) 
+    if config_file:                                                            
+        logging.config.fileConfig(config_file, disable_existing_loggers=False) 
```

in `ocrd_utils/ocrd_utils/logging.py`?

Reply:
> And you're using the `ocrd_logging.conf`? These must be set in the config file as well, the `LOGGING_DEFAULTS` are only set for the builtin logging (i.e. no ocrd_logging.conf).

Good point. I indeed use a logging config and they were not set there. Will test again.

Reply:
Resolved in eebc5cda683e9c41a76c27cb9940a95b96113891.